press-this.php

```diff
@@ -767,20 +767,20 @@ public function serve_app_html() {
 
 			<?php if ( $supports_formats ) : ?>
 				<div class="setting-modal">
-					<a href="#" class="modal-close"><span class="dashicons dashicons-arrow-left-alt2"></span><span class="setting-title"><?php _e('Post format'); ?></span></a>
+					<a href="#" class="modal-close" tabindex="-1"><span class="dashicons dashicons-arrow-left-alt2"></span><span class="setting-title"><?php _e('Post format'); ?></span></a>
```

Comment:
These are toggled with JS. The reason: while they are hidden, if you leave them on the tabindex, you end up tabbing:

 - `.post-option` (Post Formats)
 - `.modal-close` (of Post Formats, but you don't see it because it's hidden, no feedback)
 - `.post-option` (Categories)
 - `.modal-close` (of Categories, but you don't see it because it's hidden, no feedback)
 - etc

I think that `.modal-close` should be in the tabindex only if you entered the section. Even for screen readers. There's nothing to close if you didn't open anything.